archive/html_css/semantic_html.md

```diff
@@ -75,7 +75,7 @@ Headings are the `<h1>` through `<h6>` elements, and like the name implies, thes
 * `<nav>`
```

Comment:
I don't think we really need to update the archived files. @wise-king-sullyman any opinion?

Reply:
Agree